core/src/main/java/org/renjin/sexp/Environment.java

```diff
@@ -221,27 +235,120 @@ public StringVector getNames() {
   public boolean bindingIsLocked(Symbol symbol) {
     return lockedBindings != null && lockedBindings.contains(symbol);
   }
-  
-  public void setVariable(String name, SEXP value) {

```

Comment:
I think in the end, the `setVariable()`  and `findVariable()` methods are public API methods, or at least close enough to warrant some caution. If anyone else has compiled against Renjin, removing these methods will break at runtime. Let's add them back in, but mark them as `@Deprecated`. For example:

```.java
/**
 * (javadac please)
 */
@Deprecated
public void setVariable(String name, SEXP value) {
    setVariableUnsafe(name, value);
 }
```